src/main/resources/application-dev.properties

```diff
@@ -1,12 +1,22 @@
 spring.datasource.url=jdbc:mysql://localhost:3306/mmotodo
 spring.datasource.username=root
-spring.datasource.password=root
+spring.datasource.password=${DATABASE_PASSWORD}
 # -- At deplyment uncomment and change flyway.url
-spring.flyway.baseline-on-migrate=true
 #spring.flyway.url=jdbc:mysql://localhost:3306/mmotodo
 #spring.flyway.user=flyway
 #spring.flyway.password=${FLYWAY_PASSWORD}
 spring.jpa.show-sql=true
 spring.jpa.properties.hibernate.format_sql=true
 spring.jpa.open-in-view=false
 spring.graphql.graphiql.enabled=true
+spring.flyway.baseline-on-migrate=true
+
+# OAuth2 Github
+spring.security.oauth2.client.registration.github.clientId=${GITHUB_CLIENT_ID}
+spring.security.oauth2.client.registration.github.clientSecret=${GITHUB_CLIENT_SECRET}
+
+# Github sign out api route
+github.api.logout.url=${GITHUB_API_LOGOUT_URL}
+
+# Loads .env file
+spring.config.import=optional:file:.env[.properties]
```

Comment:
Could you in the pull request provide instructions on how to get these values so we can run this locally?

Reply:
Don't know if I understand? 

Do you mean to provide a the step to step guide for collaborators to get their own values so they can test on their own OAuth app?🤔

Reply:
Yeah, if anyone else wants to run your code on their local machine they need to insert these values, or the app won't run. So anyone else working on this project needs some instructions on how to make it work for themselves.